### DISCUSSION TITLE
Recommendations on Analytics tool that support tvOS?

### DISCUSSION THREAD

**Discussion by arvinsingla** [react-native-tvos/react-native-tvos#652] [Q&A] [answered]
As the title says I'm looking to see if there are folks who can recommend an analytics platform that has support for react-native-tvos. Any guidance or recommendation would be greatly appreciated.

I would also be curious what people's approaches are to error reporting?

**Comment by douglowder** [comment, marked as answer]
I do believe that Firebase analytics supports TV.
